internal/constants.go

```diff
@@ -20,7 +20,7 @@ const (
 	BootTimeoutDefault   = 30000
 	ClientMonitorDefault = 15000
 	ConfigFileName       = "configuration.toml"
-	ConfigRegistryStem   = "edgex/core/1.0/"
+	ConfigRegistryStem   = "edgex/services/1.0/"
```

Comment:
Maybe `"edgex/functions/1.0/"`? I think `services` is too generic. They're all services after all.

Reply:
Yeah, I was going off the main diagram on the homepage of edgexfoundry.org.  I can get behind `functions`, but what about `edgex/appservices/1.0`?